packages/app/src/cli/services/dev/urls.ts

```diff
@@ -151,3 +150,16 @@ export async function shouldOrPromptUpdateURLs(options: ShouldOrPromptUpdateURLs
   }
   return shouldUpdate
 }
+
+function mapRunTunnelPluginError(tunnelPluginError: plugins.TunnelPluginError) {
```

Comment:
* I'm going to change hardcoded `ngrok` for the value of the requested provider
* x should be a url to a list of available plugins right?, what should we use right now?

Reply:
I think it's fine for now since we don't have a published list of tunnel plugins yet. Once we have it, we can adjust the message to add an hyperlink to it.